src/fontra/client/web-components/range-slider.js

```diff
@@ -224,6 +226,27 @@ export class RangeSlider extends LitElement {
     `;
   }
 
+  handleKeyDown(event) {
+    event.preventDefault();
```

Comment:
We should _only_ do `preventDefault()` if we will handle the event. I think the switch needs a default branch which will simply return, and the preventDefault should be done after the switch.

Reply:
Perhaps we should return even earlier in case the control key, meta key or alt key are pressed.

Reply:
Numeric input is currently completely broken.

Reply:
Yes, I missed that @justvanrossum.

Reply:
Please implement it as I suggested:
- Return immediately if the control key, meta key or alt key are pressed
- Add a "default" branch in the switch, in which we return

Then, the `newValue !== undefined` condition is no longer needed, too.